### ISSUE TITLE
Minutes model should not save records without dates

### ISSUE THREAD

**Issue by PaulCongdon** [jlm/maint#66]
Looks like the validator in the minutes model should be updated since we aren't 'importing' minutes anymore and it appears we have minutes in the database without dates.   The validator below in models/minute.rb should be updated by dropping the unless (perhaps?):

class Minute < ActiveRecord::Base
	belongs_to :item
	belongs_to :meeting
	belongs_to :minst
	validates :date, presence: true, unless: "Rails.application.config.importing"
end

**Comment by jlm** [comment]
I don't think this would affect anything, because `importing` is only set during an import. I'd prefer to keep the option of importing, because I still want other groups to start using the app.  I'll check whether item creation via the REST interface includes a check against a nil date.
